pkg/core/chain/chain.go

```diff
@@ -384,11 +386,13 @@ func (c *Chain) ProcessSyncTimerExpired(strPeerAddr string) error {
 // acceptSuccessiveBlock will accept a block which directly follows the chain
 // tip, and advertises it to the node's peers.
 func (c *Chain) acceptSuccessiveBlock(blk block.Block, kadcastHeight byte) error {
-	startTime := time.Now().UnixMilli()
-
 	log.WithField("height", blk.Header.Height).Trace("accepting succeeding block")
 
-	prevBlockTimestamp := c.tip.Header.Timestamp
+	// TODO: Verify Certificate
```

Comment:
Is this a leftover? If not, a follow up issue would fit the case

Reply:
Good catch. I'll file it

Reply:
FYI https://github.com/dusk-network/dusk-blockchain/issues/1446